app/api/SkyChatClient.ts

```diff
@@ -215,6 +220,11 @@ export class SkyChatClient extends EventEmitter {
 
         // URL
         this.url = url;
+
+        // Plugin helpers
+        this.plugins = {
+            mute: new MutePluginHelper(this),
+        };
```

Comment:
Should be the pattern used starting now for plugins to "inject" their related logic into the client